.gitignore

```diff
@@ -106,6 +106,8 @@ venv.bak/
 # Pycharm
 .idea/
 
+# VSC
```

Comment:
manual change

Reply:
What is your preferred setup (extensions and settings) to make vscode play nicely with the pre-commit checks? :)

Reply:
i could adjust readme with vsc setup section, but preferably in a separate PR as i don't like to merge manual and automated changes together, will do